src/status_im/contexts/wallet/create_account/new_keypair/keypair_name/view.cljs

```diff
@@ -24,9 +33,8 @@
            :footer [quo/bottom-actions
                     {:actions          :one-action
                      :button-one-label (i18n/label :t/continue)
-                     :button-one-props {:disabled?           (or (zero? (count @keypair-name))
-                                                                 (> (count @keypair-name)
-                                                                    keypair-name-max-length))
+                     :button-one-props {:disabled?           (or (pos? error?)
+                                                                 (< (count @keypair-name) 4))
```

Comment:
why just `4` here? how come it's not some constant that defines the use as it was before? 🤔